test/validators.js

```diff
@@ -11621,6 +11621,20 @@ describe('Validators', () => {
         '1234577A',
         '1234577JA'],
     });
+    test({
+      validator: 'isTaxID',
+      args: ['en-IN'],
+      valid: [
+        'AAAAA1111A',
+        'BBBBB1111B',
+        'CCCCC1111C',
+        'FFFFF1111F'],
```

Comment:
```suggestion
        'FFFFF0001F',
        'ZZZFZ9999Z'],
```

Nice to make sure that the entire number range is correctly validated :)

Reply:
@braaar I have updated the test cases for entire number range validation.